.github/find-hotfix-tags.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+tag_regex='v[0-9]+\.[0-9]+\.[0-9]+-osghotfix-[A-Za-z]+'
+git tag -l | sort | egrep $tag_regex > git_tags
```

Comment:
```suggestion
git tag -l | sort | egrep -x "$tag_regex" > git_tags
```

Technically if you do not quote `$tag_regex`, it will expand to any files that match it, interpreted as a shell glob (eg, a file named `v1+.2+.3+-osghotfix-A+`).  Maybe not likely but since the variable contains glob characters that you do not want interpreted (`?`, `*`, `[]`), it's best to quote it.

Passing `-x` to (e)grep will also require that it match the whole line/tag (as opposed to any part of it), in case you care about that.